src/configuration.ts

```diff
@@ -1,24 +1,41 @@
 import { workspace } from 'vscode';
 import * as path from 'path';
 import { existsSync } from 'fs';
+import semver = require('semver');
+import { execSync } from 'child_process';
+import { outputChannel } from './extension';
 
 export interface AmebaConfig {
     command: string;
     configFileName: string;
     onSave: boolean;
+    onType: boolean;
 }
 
 export function getConfig(): AmebaConfig {
     let command = 'ameba';
     const root = workspace.workspaceFolders || [];
     if (root.length) {
         const localAmebaPath = path.join(root[0].uri.fsPath, 'bin', 'ameba');
-        if (existsSync(localAmebaPath)) command = localAmebaPath;
+        if (existsSync(localAmebaPath)) {
+            outputChannel.appendLine(`[Config] Using local ameba at ${localAmebaPath}`)
+            command = localAmebaPath;
+        } else {
+            outputChannel.appendLine(`[Config] Using system ameba`)
+        }
     }
 
+    const workspaceConfig = workspace.getConfiguration('crystal-ameba');
+    const currentVersion = execSync(`"${command}" --version`).toString();
+
+    // Support added in ameba v1.6.3
+    let onType = workspaceConfig.get<boolean>("onType", true) &&
+        semver.satisfies(currentVersion, ">=1.6.3")
```

Comment:
It was added in [v1.6.2](https://github.com/crystal-ameba/ameba/releases/tag/v1.6.2) actually.

```suggestion
    // Support added in ameba v1.6.2
    let onType = workspaceConfig.get<boolean>("onType", true) &&
        semver.satisfies(currentVersion, ">=1.6.2")
```